test failure related?

```
======================================================================
FAIL: Tests that the diffs of the template files match what we have
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/mnt/commcare-hq-ro/corehq/apps/app_manager/tests/test_app_manager_v2_diffs.py", line 167, in test_partial_02_diffs_match
    self.partial_v2_dir
  File "/mnt/commcare-hq-ro/corehq/apps/app_manager/tests/test_app_manager_v2_diffs.py", line 115, in _test_diffs_util
    test_file, filename_v1, filename_v2
AssertionError: 
*************************************
APP MANAGER V2 Diff Failure
An edit made to a V1 App Manager file "build_errors.html" does not match the stored diff
of its V2 counterpart.
**Please make the edits to the V2 file so that it gets the changes from V1.**
Once you have done this, run ./manage.py build_app_manager_v2_diffs
to rebuild the broken diffs.
If the changes are so far off that this process is confusing, please chat with
Biyeun or Jenny. Apologies in advance, if that is the case.
These files are located in
/mnt/commcare-hq-ro/corehq/apps/app_manager/tests/../templates/app_manager/v1/partials/build_errors.html
/mnt/commcare-hq-ro/corehq/apps/app_manager/tests/../templates/app_manager/v2/partials/build_errors.html
**************************************
```

--- a/corehq/apps/app_manager/tests/data/v2_diffs/partials/build_errors.html.diff.txt
+++ b/corehq/apps/app_manager/tests/data/v2_diffs/partials/build_errors.html.diff.txt
@@ -1,6 +1,6 @@
 --- 
 +++ 
-@@ -16,62 +16,56 @@
+@@ -16,68 +16,62 @@
                              {% if not error.message %}
                                  Blank form
                              {% endif %}
@@ -16,6 +16,13 @@
                              {{ error.validation_message|linebreaksbr }} in form
 -                            {% include "app_manager/v1/partials/form_error_message.html" %}
 +                            {% include "app_manager/v2/partials/form_error_message.html" %}
+                         {% case "missing instances" %}
+                             {% blocktrans %}
+                             The form is missing some instance declarations:<br/>
+                             {% endblocktrans %}
+                             {{ error.validation_message }}
+-                            {% if not not_actual_build %}in form {% include "app_manager/v1/partials/form_error_message.html" %}{% endif %}
++                            {% if not not_actual_build %}in form {% include "app_manager/v2/partials/form_error_message.html" %}{% endif %}
                          {% case "no form links" %}
                              {% blocktrans %}
                              Link to other form is selected, but no form links have been provided.
@@ -71,7 +78,7 @@
                              <a href="{% url "view_module" domain app.id error.module.id %}">{{ error.module.name|trans:langs }}</a>
                              uses cases but doesn't have a
                              case type defined.
-@@ -80,29 +74,28 @@
+@@ -86,29 +80,28 @@
                              {% url "view_form" domain app.id error.module.id error.form.id as form_url %}
                              {% blocktrans with module_name=error.module.name|trans:langs form_name=error.form.name|trans:langs %}
                              You have selected the <a href="{{ form_url }}">{{ form_name }}</a> form
@@ -108,7 +115,7 @@
                              {% endblocktrans %}
                          {% case "form filter has xpath error" %}
                              {% url "view_module" domain app.id error.module.id as module_url %}
-@@ -110,72 +103,72 @@
+@@ -116,72 +109,72 @@
                              {% blocktrans with module_name=error.module.name|trans:langs form_name=error.form.name|trans:langs %}
                              Display Condition has syntax errors
                              in the <a href="{{ form_url }}">{{ form_name }}</a> form
@@ -206,7 +213,7 @@
                              <a href="{% url "view_module" domain app.id error.module.id %}">{{ error.module.name|trans:langs }}</a>
                              has an invalid case tile configuration. Reason: {{ error.reason }}
                          {% case "no source module id" %}
-@@ -183,27 +176,25 @@
+@@ -189,27 +182,25 @@
                              <a href="{% url "view_module" domain app.id error.module.id %}">{{ error.module.name|trans:langs }}</a>
                              doesn't have a source module specified.
                          {% case "no parent select id" %}
@@ -241,7 +248,7 @@
                              {% for lang in error.missing_languages %}
                                  {{ lang }}
                              {% endfor %}
-@@ -212,20 +203,20 @@
+@@ -218,20 +209,20 @@
                          {% case "update_case uses reserved word" %}
                              Case Update uses reserved word "{{ error.word }}"
                              {% if error.case_tag %} for action "{{ error.case_tag }}"{% endif %}
@@ -266,7 +273,7 @@
                                  references a question that no longer exists: "{{ error.path }}". It is likely that
                                  this question has been renamed or deleted. Please update or delete this question
                                  reference before you make a new version.
-@@ -240,40 +231,40 @@
+@@ -246,40 +237,40 @@
                              One of your languages is empty. Check your <a href="{% url "view_app" domain app.id %}">app settings</a>.
                          {% case "missing parent tag" %}
                              A subcase is referencing a parent case tag that does not exist: "{{ error.case_tag }}"

